content/docs/hooks-overview.md

```diff
@@ -229,19 +229,19 @@ function FriendListItem(props) {
 }
 ```
 
-The state of these components is completely independent. Hooks are a way to reuse *stateful logic*, not state itself. In fact, each *call* to a Hook has a completely isolated state -- so you can even use the same custom Hook twice in one component.
+इन कौम्पोनॅन्टस की स्टेट पूरी तरह से स्वतंत्र है। Hooks *स्टेट फुल लॉजिक* का पुन: उपयोग करने का एक तरीका है, न कि state का। वास्तव में, एक Hook करने के लिए प्रत्येक *कॉल* में एक पूरी तरह से अलग state है -- जिस से आप एक ही कौम्पोनॅन्ट में दो बार एक कस्टम Hook का उपयोग कर सकते हैं।
 
-Custom Hooks are more of a convention than a feature. If a function's name starts with "`use`" and it calls other Hooks, we say it is a custom Hook. The `useSomething` naming convention is how our linter plugin is able to find bugs in the code using Hooks.
+कस्टम Hook एक विशेषता की तुलना में एक कन्वेंशन अधिक हैं। यदि किसी फ़ंक्शन का नाम "` use `" से शुरू होता है और इसे अन्य Hook कॉल करते है, तो हम कहते हैं कि यह एक कस्टम Hook है। `UseSomething` नामकरण कन्वेंशन हमारे लिंटर प्लगइन, जो Hook का उपयोग करता है, कोड में बग ढूंढने में मदद करता है।
 
-You can write custom Hooks that cover a wide range of use cases like form handling, animation, declarative subscriptions, timers, and probably many more we haven't considered. We are excited to see what custom Hooks the React community will come up with.
+आप कस्टम Hook लिख सकते हैं जो फॉर्म हैंडलिंग, एनीमेशन, डेक्लेरेटिव सब्सक्रिप्शन, टाइमर और शायद और अधिक मामलों मे जिसपे हमने विचार नहीं किया, जैसे उपयोग मामलों की एक विस्तृत श्रृंखला को कवर करते हैं। हम यह देखने के लिए उत्साहित हैं कि React समुदाय और कौन कौन से कस्टम Hooks लेकर आएंगे।
 
->Detailed Explanation
+>विस्तृत विवरण
 >
->You can learn more about custom Hooks on a dedicated page: [Building Your Own Hooks](/docs/hooks-custom.html).
+>आप एक समर्पित पेज पर कस्टम Hook के बारे में अधिक जान सकते हैं: [अपने खुद के Hooks का निर्माण](/docs/hooks-custom.html).
 
-## 🔌 Other Hooks {#other-hooks}
+## 🔌 अन्य Hooks {#other-hooks}
 
-There are a few less commonly used built-in Hooks that you might find useful. For example, [`useContext`](/docs/hooks-reference.html#usecontext) lets you subscribe to React context without introducing nesting:
+कुछ कम इस्तेमाल किए जाने वाले बिल्ड इन Hooks हैं जो आपको उपयोगी लग सकते हैं। उदाहरण के लिए, [`useContext`](/docs/hooks-reference.html#usecontext) यह आपको नेस्टिंग के बिना React context की सदस्यता लेने देता है:
```

Comment:
`की सदस्यता लेने` => `को सब्सक्राइब करने`

Reply:
<img width="158" alt="Screen Shot 2019-12-10 at 12 14 28 PM" src="https://user-images.githubusercontent.com/4654382/70502457-ca111800-1b46-11ea-858f-c064c1e0e9fc.png">

there is one issue with `करनेे` I can see two "maatras" it should be `करने`